components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -343,9 +324,8 @@ function CustomLeague:_createGameCell(args)
 
 	local content = ''
 
-	local gameData = CustomLeague.getGame()
-	if gameData then
-		content = content .. Page.makeInternalLink({}, gameData.name, gameData.link)
+	if Table.isNotEmpty(_game) then
```

Comment:
wouldn't you also want to adjust the empty check a few lines above?
`if String.isEmpty(args.game) and String.isEmpty(args.patch) then`
-->
`if Table.isEmpty(_game) and String.isEmpty(args.patch) then`